Resources/Locale/en-US/visual_effects.ftl

```diff
@@ -0,0 +1 @@
+stealth-visual-effect = [color=lightslategray]{CAPITALIZE(SUBJECT($target))} shimmers with strange transparency.[/color]
```

Comment:
```suggestion
stealth-visual-effect = [color=lightslategray]{CAPITALIZE(SUBJECT($target))} {CONJUGATE-BASIC($target, "shimmer", "shimmers")} with strange transparency.[/color]
```

this might be wrong syntax i dont remember